src/diffusers/pipeline_utils.py

```diff
@@ -71,6 +71,7 @@
         "SchedulerMixin": ["save_pretrained", "from_pretrained"],
         "DiffusionPipeline": ["save_pretrained", "from_pretrained"],
         "OnnxRuntimeModel": ["save_pretrained", "from_pretrained"],
+        "ConfigMixin": ["save_pretrained", "from_pretrained"],
```

Comment:
```suggestion
```

Could we instead have `Mel` inherit from `ModelMixin`?

Reply:
I am trying to get Mel to work inheriting from `ModelMixin`, but it feels quite hacky... `ModelMixin` expects to find a parameter from which it can infer the device and bombs out with a `StopIteration` exception while looking for one.

I realize adding a new `LOADABLE_CLASS` is a change that needs careful consideration but, to my mind, it makes sense to be able to `load_pretrained` and `save_pretrained`  a class that just has config params. The `transformers` library appears to have a class for just that kind of purpose (`ProcessorMixin`) but I thought it would be even more ugly to inherit from that.

Reply:
See what you think of the solution I came up with. Instead, I am inheriting from `SchedulerMixin`. As well as being a `LOADABLE_CLASS` this also means I can avoid the copy & paste of the `from_pretrained` and `save_pretrained`. The name is a bit counterintuitive but it seems like the best solution for now. At some point, maybe we could think of refactoring so that it is easier to create a component like `Mel` that has no `pytorch` dependencies. This has been the most tricky part of integrating the pipeline (now that you have added the optional components functionality which is very much welcomed).